### ISSUE TITLE
ability to alphabetize lists/author pages

### ISSUE THREAD

**Issue by BrittanyBunk** [internetarchive/openlibrary#2796]
there's no capability at the moment.

**Comment by mekarpeles** [comment]
@BrittanyBunk what does it mean to alphabetize lists/authors pages? i.e. alphabetize the books in author pages? The items on list pages? The issue description requires more detail.

**Comment by BrittanyBunk** [comment]
@mekarpeles right, so let's take https://openlibrary.org/authors/OL9388A/William_Shakespeare for example (I mean, it's an example - as people will say not authors are this long). To find the work I'm looking for through this page would require me to click through every page # until I find it (which is really difficult when there's so many: almost 9000 on this author alone).

That is why I'd propose a 'sort a->z' to make it much easier to not miss the book that's being looked for: 
![image](https://user-images.githubusercontent.com/55302685/71636496-0b489500-2be6-11ea-8e20-631291a77a2c.png)

This is also related to lists too (especially them, as some are around 11k works!). #2723.

**Comment by Evazisouli** [comment]
@BrittanyBunk, @jdlrobson could I work on this issue?

**Comment by jdlrobson** [comment]
@Evazisouli I'm currently trying to focus on tasks with priority 0, 1 or 2. Would gladly provide review for any of the following:
https://github.com/internetarchive/openlibrary/issues?q=is%3Aopen+label%3A%22Lead%3A+%40jdlrobson%22+-label%3A%22Priority%3A+3%22++-label%3A%22Priority%3A+4%22+

**Comment by RayBB** [comment]
I've updated this issue to reflect the current status of only needing to changing the UI and not Solr.

**Comment by davidscotson** [comment]
The `?sort=title workaround` is very handy, and I'm pleasantly surprised that it appears to be doing a multi-lingual title sort, ignoring "the", "el" and others at the start, which makes it much easier to find duplicate works.

I'm assuming this is being done automatically by solr, but had noticed that the title_sort values returned in search.json don't seem to reflect this. Is that a bug?

https://openlibrary.org/search.json?q=the+wind+in+the+willows&fields=title_suggest,title_sort

**Comment by cdrini** [comment]
Note this is now more complicated I'm afraid since we sort by the best matching edition title as opposed to the work title. We'll need an approach like this perhaps: https://blog.griddynamics.com/how-to-sort-parent-documents-by-child-attributes-in-solr/
